filters/filter.go

```diff
@@ -56,11 +56,32 @@ type Filter interface {
 }
 
 // NewFilter supports startsWith, endsWith, contains and a single wildcard
-// TODO(martinm): add rest of glob matching support and negation
+// along with negation
+// TODO(martinm): add rest of glob matching support
```

Comment:
Possibly, I was thinking about the case where I'd like to match metrics against "tagname1:tagvalue1=foo*||bar* tagname2:tagvalue2=baz*&&*w*".
How would the users specify that in the rule as a multi-filter?